CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.0-beta.22
+
+Removes external right margin of [MegaButton component](./src/components/MegaButton/MegaButton.mdx).
```

Comment:
In the future please add a level 3 heading of either "Bug Fixes, Features, or Breaking Changes" so it's clear what the impact is of the change(s)

Reply:
oops, sorry, noted